Obedient-Lobsters/WorkforceManagement
# Implemented employee list/index view

# Description
Added Employee Index view that shows Employee First name, Employee Last Name, and Department name.
 Fixes #7 
 ## Type of change
 Please delete options that are not relevant.
 # Testing Instructions

Start the program by cd'ing into the WorkforceManagement and using the command dotnet run. Once the program is running, navigate to your browser and go to the localhost address shown in the command prompt that opens up when you run the program. Click on the Employees tab in the navbar, and you should see a list with First name, Last Name, and department name for the number of employees in your database.
 # Checklist:
